src/screens/SiteDetailScreen.js

```diff
@@ -133,6 +201,7 @@ class SiteDetailScreen extends Component {
                         titleStyle={mainTitleStyle}
                         image={siteImageData ? {uri: `data:image/png;base64,${siteImageData}`} : require('../../assets/starTent.jpg')}
                         imageProps={{resizeMode: 'cover'}}
+                        imageStyle={{height: 250}}
```

Comment:
Explicitly set the height of the image in the site detail card. It was too tiny before and we couldn't really see much of the site. This is especially necessary until we have the ability for a user to click on the image for a zoomed-in view.